packages/core/useTransition/index.ts

```diff
@@ -161,14 +173,13 @@ export function useTransition(
   // current transition values
   let currentDuration: number
   let diffVector: number[]
-  let endAt: number
   let startAt: number
   let startVector: number[]
 
   // transition loop
   const { resume, pause } = useRafFn(() => {
     const now = Date.now()
-    const progress = clamp(1 - ((endAt - now) / currentDuration), 0, 1)
+    const progress = clamp( ( now - startAt ) / currentDuration ), 0, 1)
```

Comment:
What's the reason on this change?

Reply:
I find that it's equivalent, so changed it.